samples/openvino/model_optimizer/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenVINO model optimizer pipeline
+
+This is an example of a one step pipeline implementation of model optimization using OpenVINO toolkit
+
+It performs graph optimization and generates Intermediate Representation model format which can be used 
+later by the Inference Engine. 
```

Comment:
This file is still in the /samples directory. Could we move it to the /contrib directory?

Reply:
samples are now also moved to contrib